src/rag/reranker/reranker.py

```diff
@@ -57,41 +57,80 @@ async def arerank(
     ) -> list[Document]:
         """
         Rerank the documents based on which documents are most relevant to the given queries.
-        The documents are first filtered based on their relevance to the queries and capped at the input limit.
-        Then, the LLM reranker is used to rerank the filtered documents and capped at the output limit.
-        Finally, the reranked documents are returned.
+
+        If the RRF filtration is enabled:
```

Comment:
does RRF stand for the reciprocal rank fusion algorithm?

Reply:
I updated the doc to make it more clear.

Reply:
Thanks!